$OpenBSD$

LC_ALL only works for single-byte locales on OpenBSD.

--- Examples/Client/svn_cmd.py.orig	Sun Dec 19 17:50:49 2010
+++ Examples/Client/svn_cmd.py	Sun Dec 19 17:52:24 2010
@@ -70,7 +70,7 @@ def initLocale():
             locale.setlocale( locale.LC_ALL, '%s.%s' % (language_code, encoding) )
         except locale.Error:
             # force a locale that will work
-            locale.setlocale( locale.LC_ALL, 'en_GB.UTF-8' )
+            locale.setlocale( locale.LC_ALL, 'C' )
 
 def fmtDateTime( t ):
     return time.strftime( '%d-%b-%Y %H:%M:%S', time.localtime( t ) )
